core/src/main/java/com/scalar/db/api/Admin.java

```diff
@@ -348,7 +348,21 @@ void repairTable(
       throws ExecutionException;
 
   /**
-   * Add a new column to an existing table. The new column cannot be a partition or clustering key
+   * Add a new column to an existing table. The new column cannot be a partition or clustering key.
+   * <br>
+   * <br>
+   * <strong>Warning :</strong> this should be executed with significant consideration as the
+   * execution time may vary greatly depending on the underlying storage. Please plan accordingly
+   * especially if the database runs in production:
+   *
+   * <ul>
+   *   <li>for Cosmos and Dynamo DB: this operation is almost instantaneous as the table schema is
+   *       not modified. Only the table metadata stored in a separated table are updated.
+   *   <li>for Cassandra : adding a column is a constant-time operation based on the amount of data
```

Comment:
Is it a constant time or linear time?
It says constant time but it also says the time is based on the amount of data, which sounds like a linear time.

Reply:
Indeed, it is ambiguous. I copied this from the official Cassandra [documentation](https://cassandra.apache.org/doc/latest/cassandra/cql/ddl.html#alter-table-statement). 
I haven't found online any other information besides this about the add column performance. 
I added a [question](https://stackoverflow.com/questions/73060026/is-altering-a-table-to-add-a-column-a-constant-time-or-linear-time-operation) on StackOverflow to ask about it.

Reply:
My StackOverflow question received a detailed [answer](https://stackoverflow.com/a/73060202/9538402) which clarified things. 

What do you think about updating the text as follows?
```suggestion
   *   <li>for Cassandra: adding a column will only update the schema metadata and do not modify existing schema records. The cluster topology is the main factor for the execution time. Since the schema metadata change propagates to each cluster node via a gossip protocol, the larger the cluster, the longer it will take for all nodes to be updated.

Reply:
Looking good. Thank you!